index.html

```diff
@@ -45,14 +46,13 @@ <h3>On fire, no bugs</h3>
 			</div>
 			
 			<section class="container">
-				<h1>About me:</h1>
+				<h1 id="aboutMe">About me:</h1>
 				<p>This is where I'm going to tell you all about myself and the skills that I am developing on the Full Stack Track at Mayden Academy.  There's the minor detail of the online test, and then I'll be a certified scrum master, which I'll tell you about too.  I'm based in Bath and am looking for my first role as a junior developer: I'm available from the middle of December 2019.</p>
```

Comment:
This font size shrinks when I go to the mobile option, making it quite difficult to read.

Reply:
Ok that's useful, I wasn't sure if i had made it too small.  I'll skip the smallest media query on the font size.